mtp_prisoner_location_admin/templates/prisoner_location_admin/location_file_upload.html

```diff
@@ -9,23 +9,27 @@
 <form action="{% url "location_file_upload" %}"
       method="post"
       enctype="multipart/form-data"
-      class="js-uploadSubmit">
+      class="upload-form">
   {% csrf_token %}
 
   {% include 'mtp_common/forms/error-summary.html' with form=form only %}
+  <div class="form-group">
+    Use this page to upload a prisoner location file in CSV format (.csv).
```

Comment:
this line ought to have a `{% trans '…' %}` wrapper, but admittedly we're pretty far away from worrying about welsh…